packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/archive/translations/Messages_fr_FR.json

```
@@ -1,8 +1,14 @@
 {
   "pci_projects_project_storages_cold_archive_containers_container_archive_title": "Archiver votre conteneur",
-  "pci_projects_project_storages_cold_archive_containers_container_archive_description": "Vous êtez sur le point d'archiver votre conteneur {{ containerName }}.",
-  "pci_projects_project_storages_cold_archive_containers_container_archive_submit_label": "Confirmer",
+  "pci_projects_project_storages_cold_archive_containers_container_archive_description": "Vous êtes sur le point d'archiver votre conteneur {{ containerName }}.",
+  "pci_projects_project_storages_cold_archive_containers_container_archive_description_2": "En configurant une durée de rétention, votre archive sera verrouillée et vous ne pourrez plus supprimer vos données pendant toute la période de rétention.",
+  "pci_projects_project_storages_cold_archive_containers_container_archive_submit_label": "Archiver",
   "pci_projects_project_storages_cold_archive_containers_container_archive_cancel_label": "Annuler",
   "pci_projects_project_storages_cold_archive_containers_container_archive_success_message": "Le conteneur {{ containerName }} est en cours d'archivage.",
-  "pci_projects_project_storages_cold_archive_containers_container_archive_error_message": "Une erreur est survenue lors de l'archivage du conteneur {{ containerName }} : {{ message }}."
+  "pci_projects_project_storages_cold_archive_containers_container_archive_error_message": "Une erreur est survenue lors de l'archivage du conteneur {{ containerName }} : {{ message }}.",
+  "pci_projects_project_storages_cold_archive_containers_container_archive_has_retention": "Planifier une durée de rétention",
+  "pci_projects_project_storages_cold_archive_containers_container_archive_has_no_retention": "Ne pas planifier de durée de rétention",
+  "pci_projects_project_storages_cold_archive_containers_container_archive_retention_days": "Ajouter le nombre de jours de rétention",
+  "pci_projects_project_storages_cold_archive_containers_container_retention_warning": "Votre durée de rétention prendra fin le <strong>{{ date }}</strong>.</br>Vous ne pourrez procéder à la suppression de vos données qu'une fois cette date passée.",
```

You can also split it into 2 keys to avoid adding break lines

```suggestion
  "pci_projects_project_storages_cold_archive_containers_container_retention_warning": "Votre durée de rétention prendra fin le <strong>{{ date }}</strong>.<br />Vous ne pourrez procéder à la suppression de vos données qu'une fois cette date passée.",
```